src/VoerroTagsInput.vue

```diff
@@ -122,6 +122,11 @@ export default {
             type: String,
             default: 'value',
         },
+        
+        displayField: {
+          type: String,
+          default: 'display',
```

Comment:
I think it should default to `null` or something, and we should check if the field is set in the `v-html` above. There might be cases when you have a field called "display" on your data which is not meant to be the `displayField` for the tags input, just a coinsidense.

Reply:
Hmm... this is an interesting thought. I certainly may be missing something as I don't have such a deep knowledge of the package as you. Let me try to explain my thought process so it's clear why I did this...

I tried to implement the same method you used for the `idField` and `textField` where the default values are `key` and `value`, respectively.  Setting the default value of `displayField` to `null` would force the user to pass a prop for the name of the `displayField`. There should be some baseline naming convention to pass through their data and then override it if needed.

Intended use something like this:
```
$data = [
  'key' => '1234',
  'value' => 'Anthony Cirinelli',
  'display' => '<img src="test.jpg"> Anthony Cirinelli <br> Job Title'
];
```

Reply:
I'm 50/50 on this. I understand your logic, but at the same time the whole `displayField` is kind of optional so it doesn't have to have a default value in theory. It's like you only specify it when you need it. The idea with custom `idField` and `textField` is that you can grab and pass your data as is without re-mapping it and then just provide the correct field names. Leaving the last word to you.